src/VolleyManagement.Backend/VolleyManagement.Services/Authentication/VolleyUserStore.cs

```diff
@@ -246,8 +246,8 @@ public Task RemoveLoginAsync(UserModel user, UserLoginInfo login)
         /// <returns>Task to await</returns>
         public Task<IList<UserLoginInfo>> GetLoginsAsync(UserModel user)
         {
-            return Task.FromResult((IList<UserLoginInfo>)user.Logins.ConvertAll(
-                l => new UserLoginInfo(l.LoginProvider, l.ProviderKey)));
+            return Task.FromResult((IList<UserLoginInfo>)((List<LoginProviderModel>)(user.Logins))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unnecessary cast to 'List<LoginProviderModel>'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1905)